packages/edit-site/src/components/sidebar-navigation-screen-patterns/index.js

```diff
@@ -118,55 +129,60 @@ export default function SidebarNavigationScreenPatterns() {
 		( select ) => select( coreStore ).getCurrentTheme()?.is_block_theme,
 		[]
 	);
-	const isTemplatePartsMode = useSelect( ( select ) => {
-		const settings = select( editSiteStore ).getSettings();
-		return !! settings.supportsTemplatePartsMode;
-	}, [] );
-
-	const templatePartsLink = useLink( {
-		path: '/wp_template_part/all',
-		// If a classic theme that supports template parts accessed
-		// the Patterns page directly, preserve that state in the URL.
-		didAccessPatternsPage:
-			! isBlockBasedTheme && isTemplatePartsMode ? 1 : undefined,
-	} );
-
-	const footer = ! isMobileViewport ? (
-		<ItemGroup>
-			{ ( isBlockBasedTheme || isTemplatePartsMode ) && (
-				<SidebarNavigationItem withChevron { ...templatePartsLink }>
-					{ __( 'Manage all template parts' ) }
-				</SidebarNavigationItem>
-			) }
-		</ItemGroup>
-	) : undefined;
 
+	/**
+	 * This sidebar needs to temporarily accomodate two different "URLs":
+	 *
+	 * 1. path = /patterns
+	 *    Block based themes. Also classic themes can access this URL, though it's not linked anywhere.
+	 *
+	 * 2. path = /wp_template_part/all
+	 *    Classic themes with support for block-template-parts. We need to list only Template Parts in this case.
+	 *    The URL is accessible from the Appearance > Template Parts menu.
+	 *
+	 * This is temporary. We aim to consolidate to /patterns.
+	 */
 	return (
 		<SidebarNavigationScreen
 			isRoot={ ! isBlockBasedTheme }
-			title={ __( 'Patterns' ) }
-			description={ __(
-				'Manage what patterns are available when editing the site.'
-			) }
-			actions={ <AddNewPattern /> }
-			footer={ footer }
+			title={
+				isTemplatePartsPath
+					? __( 'Manage template parts' )
+					: __( 'Patterns' )
+			}
+			description={
+				isTemplatePartsPath
+					? __(
+							'Create new template parts, or reset any customizations made to the template parts supplied by your theme.'
+					  )
+					: __(
+							'Manage what patterns are available when editing the site.'
+					  )
+			}
+			actions={
+				<AddNewPattern
+					canCreateParts={ isBlockBasedTheme }
+					canCreatePatterns={ ! isTemplatePartsPath }
+				/>
+			}
```

Comment:
```suggestion
			actions={
				( isBlockBasedTheme || ! isTemplatePartsPath ) && (
					<AddNewPattern
						canCreateParts={ isBlockBasedTheme }
						canCreatePatterns={ ! isTemplatePartsPath }
					/>
				)
			}
```

How about changing to conditional rendering? Otherwise, the hidden field will be rendered even though there are no controls.

![template-parts-no-control](https://github.com/WordPress/gutenberg/assets/54422211/7be76489-6a93-4329-86a9-de997e4dd5ee)

Reply:
Done at [08e8ebe](https://github.com/WordPress/gutenberg/pull/60689/commits/08e8ebeb81985a8512f6861c858516adda5ba04b)